ui/src/map/layers/operational-point/operational-points-layer-utils.ts

```diff
@@ -171,8 +176,50 @@ export const renderOperationalPointFeature = (
     });
     feature.set(OPERATIONAL_POINT_FEATURE_DATA_PROPERTY, point);
     feature.setStyle((feature, resolution) =>
-        getOperationalPointStyleForFeature(feature, resolution, featureMode),
+        getOperationalPointCircleStyleForFeature(feature, resolution, featureMode),
     );
 
     return feature;
 };
+
+export const operationalPointAreaPolygonStyle = function (isNew: boolean) {
+    return function (feature: Feature<OlPolygon>) {
+        const coords = getFeatureCoords(feature);
+        const isValid = isValidPolygon(coords, isNew);
+        const lineColor = isValid ? '#009BFF' : 'red';
+        const fillColor = isValid ? '#009BFF35' : 'rgba(255, 150, 0, 0.1)';
```

Comment:
Otin nämä värit suoraan Karin pocista. Ne eivät mäppäydy oikeastaan mihinkään Geoviitteen nykyväreihin, mutta ovat "sinnepäin". Oletan että Karilla pocissaan oli se idea, että kun kyseessä on hieman eri värisävyt kun esim. valituilla sijaintiraiteilla, niin värien merkitys säilyy mutta erilaiset viivat erottaa paremmin toisistaan.